plugins/kubernetes/README.md

```diff
@@ -185,6 +185,8 @@ Samson automatically adds `container[].lifecycle.preStop` `sleep 3` if a preStop
 `container[].samson/preStop` is not set to `disabled`, to prevent in-flight requests from getting lost when taking a pod
 out of rotation (alternatively set `metadata.annoations.container-nameofcontainer-samson/preStop: disabled`).
 
+The default behavior can be to never add this by setting the `KUBERNETES_DONT_ADD_PRESTOP=false`
```

Comment:
better to let people opt-in since this is kinda hacky anyway so only do it if`KUBERNETES_ADD_PRESTOP=true`

Reply:
Oh, will do it that way. Thanks!

Reply:
I didn't want to force changes on your side =)